docs/guides/installation.md

```diff
@@ -23,23 +23,22 @@ The knative project is a dependency of TriggerMesh, install it using the instruc
 All TriggerMersh APIs are implemented as Kubernetes CRDs, which we need to create before deploying the controller. The following `kubectl apply` command will create all CRDs.
 
 ```
-kubectl apply -f https://github.com/triggermesh/triggermesh/releases/download/v1.10.1-rc1/triggermesh-crds.yaml
+$ kubectl apply -f https://github.com/triggermesh/triggermesh/releases/download/v1.10.1-rc1/triggermesh-crds.yaml
 ```
 
 ## Install the controller
 
 By default the controller gets deployed in the `triggermesh` namespace. Deploy the controller with the following `kubectl apply` command:
 
 ```
-kubectl apply -f https://github.com/triggermesh/triggermesh/releases/download/v1.10.1-rc1/triggermesh.yaml
+$ kubectl apply -f https://github.com/triggermesh/triggermesh/releases/download/v1.10.1-rc1/triggermesh.yaml
 ```
 
 ## Verifying the installation
 
 Upon successful creation of the CRDs and successful deployment of the controller you should see two Pods running in the `triggermesh` namespace
 
 ```
```

Comment:
It would help to set this to ```` ```console````.
Anything that involves a command followed by some output should be colorized accordingly.

Example:

```console
$ kubectl get pods -n triggermesh
NAME                                                   READY   STATUS    RESTARTS   AGE
triggermesh-controller-5cd97f4c8f-z6r2r                1/1     Running   0          57m
```